docs/preview/connected-registry/connected-registry-error-codes.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Connected registry error code reference
+description: Details about error codes shown in the statusDetails property of a connected registry resource. For each error, possible solutions are listed.
+ms.topic: article
```

Comment:
(minor) You can use use `ms.topic: troubleshooting` when this is published to docs

Reply:
Thanks @dlepow. Updated this.